src/Stack/Dot.hs

```diff
@@ -140,28 +140,37 @@ listDependencies opts = do
   if listDepsTree opts then
     do
       liftIO $ Text.putStrLn "Packages"
-      liftIO $ printTree opts 0 [] pkgs resultGraph
```

Comment:
So the current line ignores the targets completely — am I reading that correctly?

Reply:
Yes, that is correct.